### ISSUE TITLE
[Suggestion] Add a way to have Hives & Expansion Boxes not connect

### ISSUE THREAD

**Issue by LuziferSenpai** [JDKDigital/productive-bees#455]
Hey,

it would be awesome to get something that prevents hives & expansion boxes to not connect or have only hives & connection boxes of the same material connect

**Comment by JaisDK** [comment]
why?

**Comment by LuziferSenpai** [comment]
Because its messes with the way to extract/input into the hive, if all the sides of the hive are blocked

**Comment by JaisDK** [comment]
you can extract from and pipe into the expansion box

**Comment by JaisDK** [comment]
even if you could only connect to the hive, how does making the expansion box not connect help in any way?

**Comment by LuziferSenpai** [comment]
Yes it does, because that way we can say that in a circle of 4 hives with 4 expansion boxes besides, that specific hives connect to specific expansion boxes

**Comment by radwadas** [comment]
I subscribe to this suggestion. It's really tricky and can be buggy to stack hives because of this behaviour. I've raised a separate issue on a possible exploit. Having a way to specify in which direction the hive should stack would make life a lot easier. At the very least, different wood types not stacking with each other would be a good workaround.
